Sources/ApolloCodegenLib/ApolloSchemaOptions.swift

```diff
@@ -47,16 +47,16 @@ public struct ApolloSchemaOptions {
       "--endpoint=\(self.endpointURL.absoluteString)"
     ]
     
-    if let header = self.header {
-      arguments.append("--header=\(header)")
-    }
-    
     if let key = self.apiKey {
       arguments.append("--key=\(key)")
     }
     
     arguments.append(outputURL.path)
     
+    if let header = self.header {
+      arguments.append("--header=\(header)")
+    }
```

Comment:
Can you throw a comment in here so next time I decide to be "efficient" and alphabetize all this stuff again, I don't put it back? 😃

Reply:
Sure!